### ISSUE TITLE
Good filename convention

### ISSUE THREAD

**Issue by gernest** [OsvaldoTCF/order2go#3]
@OsvaldoTCF @euller88   Fantastic work. Just want to tell you that you can use underscores for filenames. Instead of `suppliercontroller.go` you can write `supplier_controller.go`.

If there is anything I can help please le me know.

**Comment by eullerpereira94** [comment]
Thank @gernest, we appreciate your sincere opinion, but for now, this project is just an early proof of concept, what you can do now is to help us improve the quality of the project, by searching for bugs and inefficient code.

**Comment by gernest** [comment]
@euller88 okay. And did you get around helen? Never heard from you ever since I fixed the problem.

**Comment by eullerpereira94** [comment]
Yes, it's working after all, but I'm not using it with the o2go project because I tried to stick with the initial propose. I was only playing with it a little bit, because after seeing how good utron is, I was checking your other project compare quality, and I can say that I'm not disappointed.
